app/assets/stylesheets/base/typography.scss

```diff
@@ -2,17 +2,18 @@
 @import '../config/import';
 
 :root {
-  --content-rythm: var(--su-2);
-  --content-font-size: var(--fs-l);
+  --content-rhythm: var(--su-2);
```

Comment:
lol

me + english = 💥

Reply:
It's a really weird word and the pronunciation/spelling isn't logical :P!